content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -8,6 +8,10 @@ layout: bundle
 
 #### Planned
 
+##### Breaking change in REST APIs
+
+As of release 10.18+, wrong value of `pageSize` or `currentPage` will result in response code of 422 instead of 500.
```

Comment:
In 10.18 announcements, it should go under "Implemented".

Reply:
I've corrected it